src/preview.js

```diff
@@ -1,51 +1,76 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
 import countries from '../assets/countries.json';
 import continentNames from '../assets/continent-names.json';
 import continents from '../assets/continents.json';
-import { getEmojiFlag } from './utils';
-import { __, sprintf } from '@wordpress/i18n';
+
+import { getEmojiFlag, formatExcerpt } from './utils';
 
 export default function Preview( { countryCode, relatedPosts } ) {
 	if ( ! countryCode ) return null;
 
 	const emojiFlag = getEmojiFlag( countryCode ),
-	      hasRelatedPosts = relatedPosts?.length > 0;
+		hasRelatedPosts = relatedPosts?.length > 0;
 
 	return (
 		<div className="xwp-country-card">
-			<div className="xwp-country-card__media" data-emoji-flag={ emojiFlag }>
-				<div className="xwp-country-card-flag">
-					{ emojiFlag }
-				</div>
+			<div
+				className="xwp-country-card__media"
+				data-emoji-flag={ emojiFlag }
+			>
+				<div className="xwp-country-card-flag">{ emojiFlag }</div>
 			</div>
 			<h3 className="xwp-country-card__heading">
-				{ __( 'Hello from' ) }
-				{ ' ' }
-				<strong>{ countries[countryCode] }</strong>
-				{ ' ' }
-				(<span className="xwp-country-card__country-code">{ countryCode }</span>),
-				{ ' ' }
-				{ continentNames[continents[countryCode]] }!
+				{ __( 'Hello from' ) }{ ' ' }
+				<strong>{ countries[ countryCode ] }</strong> (
+				<span
+					className="xwp-country-card__country-code"
+					data-country-code={ countryCode }
+				>
+					<abbr title={ countries[ countryCode ] }>
+						{ countryCode }
+					</abbr>
+				</span>
+				), { continentNames[ continents[ countryCode ] ] }!
 			</h3>
```

Comment:
@cdevroe Do you see potential i18n issues with the card heading? What prevented you from making the string easier to localize? Do you have an idea of how this should be handled?

Reply:
Yes I do! And I totally missed that these strings didn't include the text-domain. So I've added those to the PR.

When I saw this code I believe my assumption was that the space between “Hello from” and the country name was to be kept separate from the text definition? But I’m unsure on that. So I do not know the proper way to handle trailing spaces in localized strings.